content/docs/pulumi-cloud/reference/deployments/_index.md

```diff
@@ -0,0 +1,1043 @@
+---
+title: Deployments
+title_tag: "Pulumi Cloud REST API: Deployments"
+meta_desc: Learn about the Pulumi Cloud REST API endpoints for configuring and managing Pulumi Deployments to automate infrastructure updates.
+menu:
+    cloud:
+        parent: pulumi-cloud-reference
+        weight: 4.5
```

Comment:
Per Hugo docs, this should be an integer. Not sure how this will sort. Likely it will interpret both this page and the other page that has `weight: 4` as being at the same weight, then do a secondary lexicographical sort? Anyway, for consistent ordering each page needs to be an integer, which does make it a PITA to insert a new page in a large list of ordered pages like this because you have to go update all of the other pages if there weren't any numerical gaps between them.

Reply:
https://gohugo.io/content-management/menus/#weight

Reply:
@thoward Are you ok with re-doing the weights as integers (multiples of 10) in a separate PR? It does display correctly for me.